process/blockProcessor.go

```diff
@@ -20,7 +20,7 @@ const (
 	internalMetaBlockByNoncePath  = "/internal/%s/metablock/by-nonce"
 	internalShardBlockByNoncePath = "/internal/%s/shardblock/by-nonce"
 
-	internalMiniBlockByHashPath = "/internal/%s/miniblock/by-hash"
+	internalMiniBlockByHashPath = "/internal/%s/miniblock/by-hash/%s/epoch"
```

Comment:
can we use `"/internal/%s/miniblock/by-hash/%s/epoch/%d"` and set the epoch as well on L277 ?

Reply:
done